test-common/src/main/daml/upgrades/RecordFieldsNewNonOptional/v2/Main.daml

```diff
@@ -0,0 +1,15 @@
+-- Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module Main where
+
+import Daml.Script
+
+main : Script ()
+main = pure ()
+
+data Struct = Struct
```

Comment:
Cmon now, this is daml! We have `with`!

Reply:
my cold dead hands

Reply:
Jokes aside, our tests should probably test the syntax we recommend to our users.

Reply:
fixed